requirements/py36.yml

```diff
@@ -16,9 +16,11 @@ dependencies:
   - esmpy>=7.0
 
 # Test dependencies.
+  - asv
   - black=20.8b1
   - flake8
   - flake8-docstrings
   - flake8-import-order
+  - nox
```

Comment:
I'm interested to know the reasoning behind why nox is now a dependancy.

Reply:
ASV requires Nox for environment management. If a developer runs `asv run ...` without Nox being installed: the run will fail. And of course a developer running `nox --session benchmarks` by definition needs Nox too. Either way, adding benchmarking makes Nox a requirement.